Lib/ufo2ft/makeotfParts.py

```diff
@@ -156,6 +154,19 @@ def setupAnchorPairs(self):
 
         self.mkmkAnchorPairs = self.anchorPairs
 
+    def _findLayoutFeatures(self):
+        """Returns what OpenType layout feature tags are present in the UFO."""
+        if self.font.path is None:
+            return set()
+        feapath = os.path.join(self.font.path, "features.fea")
+        if not os.path.exists(feapath):
+            return set()
+        glyphMap = self.outline.getReverseGlyphMap()
+        parser = feaLib.parser.Parser(feapath, glyphMap=glyphMap)
```

Comment:
Not re-parsing might save a few milliseconds (on my laptop, parsing and building the feaLib test cases takes less than 1ms per input file on average; although these are rather small test). But feaLib would need a different API, so I think it's not worth the trouble. If performance turns out to be a problem, we can always change this later. Other parts of the pipeline are a much bigger bottleneck.